docs/data/toolpad/building-ui/canvas-and-inspector.md

```diff
@@ -0,0 +1,43 @@
+# Canvas
+
+<p class="description">
+    Canvas and Inspector are the two main areas when it comes to UI building
+</p>
+
+## Working with canvas
+
+Building UI is as easy as dragging component from the library and dropping it on the canvas.
+
+Once you have a few components on the canvas you can easily tweak them by doing following actions:
+
+<img src="/static/toolpad/docs/canvas-controls.png" width="600px" alt="Canvas controls" />
+
+- Select component - **click** on any component
+- Changing component's order/placement on the canvas - initiate draggin by **clicking** on the component's hud when you see mouse cursor change to a hand icon
+- Remove component from canvas - **click** bin icon in the component's hud
+
+## Inspector
```

Comment:
What do you think about removing the previous 3 bulletpoints and instead create 3 sections instead of the `## Inspector` one:

```md
## Selecting elements

Click any elements in the canvas to select it. An overlay indicates the selected elements. In the inspector panel you can see the values of the selected elelements's properties. Each elements has a specific set of properties that you can tweak in the inspector panel.

<img src="/static/toolpad/docs/inspector.png" width="270px" alt="Inspector" />

You can make some properties dynamic by binding them to javascript expressions. Such properties have a link icon to the right of their control. Read the [data binding](/toolpad/data-binding/) section to find out more about this topic.

## Removing elements

Select the elements you want to remove. Click the bin icon in the overlay to remove it from the canvas. You can also press the <kbd class="key">Backspace</kbd> key to achieve the same result.

## Moving elements around

To move elements around on the canvas, select it and drag it from the overlay handle to its new location. The canvas will highlight available drop locations. Drag it to where you want to move it to and release the mouse button to permanently move it to the new location.
```

Additionally, should we make a difference between components (like classes) and elements (like instances) when we're talking about the component as a model (in the component library) or when it's instantiated in the page. Or is it fine to refer to both of those as "components". (I experimented with using "element" in the above suggestion, but perhaps it will be too confusing? Do users benefit from having these two concepts distinctly named?)

Reply:
I wouldn't use components/elements, instead if we really need to we could refer to **component** and **component instance** 🤔

I'm trying to think from a user perspective right now and to me it doesn't seem to make much difference component vs instance, i.e.:
1. I take component from library and drag it onto canvas
2. I change that component's properties in inspector

Even though it's not component's class that I'm working with to me that is still the same component 🤔

Reply:
sure